apps/system/test/unit/mock_places.js

```diff
@@ -4,6 +4,32 @@
 function MockPlaces() {}
 
 MockPlaces.prototype = {
+  init: function() {
+    this._store = {};
+  },
   start: function() {},
+
+  defaultPlace: function(url) {
+    return {
+      url: url,
+      title: url,
+      icons: {},
+      frecency: 0,
```

Comment:
https://github.com/weilonge/gaia/blob/syncto.poc/apps/system/js/places.js#L197

I think that means Frecency algorithm.
https://developer.mozilla.org/en-US/docs/Mozilla/Tech/Places/Frecency_algorithm